doc/netplan.md

```diff
@@ -317,44 +317,73 @@ similar to ``gateway*``, and ``search:`` is a list of search domains.
 :   Configure policy routing for the device; see the ``Routing`` section below.
 
 ## DHCP Overrides
-Several DHCP behavior overrides are available. Currently this is only supported
-via the ``networkd`` backend.
+Several DHCP behavior overrides are available. Most currently only have any
+effect when using the ``networkd`` backend, with the exception of ``use-routes``
+and ``route-metric``.
 
 Overrides only have an effect if the corresponding ``dhcp4`` or ``dhcp6`` is
 set to ``true``.
 
 If both ``dhcp4`` and ``dhcp6`` are ``true``, the ``networkd`` backend requires
-that ``dhcp4-overrides`` and ``dhcp6-overrides`` contain the same keys and values.
+that ``dhcp4-overrides`` and ``dhcp6-overrides`` contain the same keys and
+values. If the values are mismatched, an error will be shown and the network
+configuration will not be applied.
+
+When using the ``NetworkManager`` backend, overrides may have mismatched values
+between ``dhcp4-overrides`` and ``dhcp6-overrides``, and will be applied to the
+DHCP client processes as specified in the netplan YAML.
 
```

Comment:
Wordsmithing:

If the values do not match, an error will be shown and the network configuration will not be applied.

When using the ``NetworkManager`` backend, different values may be specified for ``dhcp4-overrides`` and ``dhcp6-overrides``, and will be applied to the DHCP client processes as specified in the netplan YAML.